src/api/app/models/scm_status_report.rb

```diff
@@ -0,0 +1,14 @@
+class ScmStatusReport < ApplicationRecord
+  belongs_to :workflow_run
+end
+
+# == Schema Information
+#
+# Table name: scm_status_reports
+#
+#  id              :integer          not null, primary key
+#  response_body   :text(4294967295)
```

Comment:
I think you also want to record what you send (request), analog to the webhooks. Like whatever is happening in `SCMStatusReporter.call`

Reply:
You mean the request payload and it's headers?